src/Microsoft.IdentityModel.Logging/IdentityModelTelemetryUtil.cs

```diff
@@ -23,6 +24,27 @@ public static class IdentityModelTelemetryUtil
             [versionTelemetry] = ClientVer
         };
 
+        // Static attribute tags
+        internal const string WilsonVersionTag = "WilsonVersion";
+        internal const string MetadataAddressTag = "MetadataAddress";
+        internal const string RefreshReasonTag = "RefreshReason";
+        internal const string OperationStatusTag = "OperationStatus";
+        internal const string ExceptionTypeTag = "ExceptionType";
+
+        // Validation result values
+        internal const string Success = "Success";
+        internal const string Failure = "Failure";
+
+        // Configuration manager refresh reasons
+        internal const string Requested = "Requested";
+        internal const string Scheduled = "Scheduled";
```

Comment:
This was a suggested value for one of the dimensions, but I'm not yet sure how to the reason for the refresh